frontend/src/app/utils/validation.ts

```diff
@@ -1,6 +1,19 @@
 export const VALIDATION_PATTERNS = {
   integerUnsigned: '^[0-9]+$',
-  withoutNumbers: '^[^0-9]+$',
-  phoneNumber: '^[\\+\\(\\)0-9\\s-]*?$',
-  email: '^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-z]{2,4}$'
+
```

Comment:
Wenn wir immer das gesamte Pattern prüfen und nur eine generische Fehlermeldung ausgeben, weiß der User im Zweifelsfall nicht, was er falsch gemacht hat bzw. welches Zeichen ungültig ist. Beim Passwortvalidator wurde das z. B. auseinandergezogen und es werden unterschiedliche Fehlermeldungen ausgegeben. War das hier nicht notwendig/nicht gewünscht oder wurde aus Zeitgründen vereinfacht?

Reply:
ich weiß was du meinst, wir ziehen das in zukunft noch mal nach